frontend/packages/dev-console/integration-tests/features/user-preferences/user-preferences-dev-perspective.feature

```diff
@@ -11,6 +11,7 @@ Feature: Manage user preferences
         Scenario: Setting Developer preference for perspective: UP-01-TC01
             Given user is at admin perspective
              When user clicks on user dropdown on masthead and selects "User Preference"
+              And user sees "General" tab selected on User Preferences page
```

Comment:
This line can not be part of `When` statements, add it in `Then` statements

Reply:
Or just remove it from here as it does not seem to be appropriate for the `Then` statements